doc/nrf/libraries/others/event_manager_proxy.rst

```diff
@@ -0,0 +1,158 @@
+.. _event_manager_proxy:
+
+Event Manager proxy
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Event Manager proxy is a library that passes events between cores.
```

Comment:
s/passes events between/passes Event Manager events between
best to use reference to event manager in `Event Manager`

Reply:
ok